R/Archive.R

```diff
@@ -49,6 +49,7 @@ Archive = R6Class("Archive",
       assert_data_table(ydt)
       assert_list(xss_trafoed)
       assert_data_table(ydt[, self$cols_y, with = FALSE], any.missing = FALSE)
+      self$search_space$assert_dt(xdt[, self$cols_x, with =FALSE])
```

Comment:
That cannot work, right? `ParamSet$assert_dt()` is only specified for ParamSets that do not have trafos, right?